packages/x11/xserver/xorg-server/package.mk

```diff
@@ -158,11 +158,6 @@ post_makeinstall_target() {
   mkdir -p $INSTALL/etc/X11
     find_file_path config/xorg.conf &&
       cp $FOUND_PATH $INSTALL/etc/X11
```

Comment:
Small problem here - if `find_file_path` exits with 1 (which it will do with a vanilla master build) then the exit code for `post_makeinstall_target()` is also 1, and the build will fail. This now needs to be:
```
   mkdir -p $INSTALL/etc/X11
     find_file_path config/xorg.conf &&
       cp $FOUND_PATH $INSTALL/etc/X11 || true
```

Or add `return 0` at the end of `post_makeinstall_target()`.

Reply:
thanks for the feedback, hadn't build-tested x86 and missed that one...

we'll have to do it properly, i.e. put find_file_path in an if, otherwise a return 0 or || true will mask problems if cp fails

I'll have a look at it